bedrock/legal/templates/legal/fraud-report.html

```diff
@@ -27,6 +27,20 @@ <h3 class="mzp-c-sidemenu-label">{{ self.page_title() }}</h3>
 {% endblock %}
 
 {% block article %}
+
+{% if form_submitted %}
+<aside class="mzp-c-notification-bar mzp-t-success">
```

Comment:
Thanks! It would be nice to add some bottom margin below the notification here:

![image](https://user-images.githubusercontent.com/400117/89388579-20495180-d6fc-11ea-8532-3595772a1d40.png)

Using `margin-bottom: $spacing-xl;` should do it:

![image](https://user-images.githubusercontent.com/400117/89388823-774f2680-d6fc-11ea-9793-95f41f0f3e5c.png)